esmvaltool/preprocessor/_derive/_derived_variable_base.py

```diff
@@ -45,20 +46,24 @@ def get_required(self, frequency):
             required variables are declared.
 
         """
-        required_variables = dict(self.__class__._required_variables)
+        required_variables = copy.deepcopy(self.__class__._required_variables)
```

Comment:
I think you can simply write
```suggestion
        required_variables = copy.deepcopy(self._required_variables)
```
or doesn't it work?

Reply:
True that, I wasn't aware of :smile:. It only makes a difference if there is an instance **and** class variable with the same name.